StyleCop.Analyzers/StyleCop.Analyzers/Settings/ObjectModel/AnalyzerConfigHelper.cs

```diff
@@ -51,6 +51,12 @@ internal static string TryGetStringValue(AnalyzerConfigOptionsWrapper analyzerCo
             return null;
         }
 
+        internal static string TryGetMultiLineStringValue(AnalyzerConfigOptionsWrapper analyzerConfigOptions, string key, bool allowExplicitUnset = true)
+        {
+            var orgValue = TryGetStringValue(analyzerConfigOptions, key, allowExplicitUnset);
+            return orgValue?.Replace("\\r", "\r")?.Replace("\\n", "\n");
```

Comment:
Only the first `?.` needs to handle null, right?

```suggestion
            return orgValue?.Replace("\\r", "\r").Replace("\\n", "\n");
```